hack/tests/ci-operator-prepare.sh

```diff
@@ -58,8 +47,18 @@ export RESOURCEGROUP_TTL=4h
 prdetail="$(python -c 'import json, os; o=json.loads(os.environ["CLONEREFS_OPTIONS"]); print "%s-%s-" % (o["refs"][0]["pulls"][0]["author"].lower(), o["refs"][0]["pulls"][0]["number"])' 2>/dev/null || true)"
 export RESOURCEGROUP="$(basename "$0" .sh)-$prdetail$(cat /dev/urandom | tr -dc 'a-z' | fold -w 6 | head -n 1)"
 
-setup_secrets
+ln -sf /usr/local/e2e-secrets/azure secrets
 
 set +x
+. ./secrets/secret
+if [[ -e /usr/local/e2e-secrets/azure ]]; then
```

Comment:
I think this is not needed as we already define/checked if we are running in CI at line 38th.

Reply:
oh yeah, duh ;)